libcodechecker/libhandlers/cmd.py

```diff
@@ -300,14 +300,6 @@ def __register_suppress(parser):
 
     group = parser.add_mutually_exclusive_group(required=True)
 
-    group.add_argument('-e', '--export',
```

Comment:
If we are changing it and updating the usage of `cmd suppress`, let's update the oneline and the multiline description later on where the subcommand is defined, and keep the `user_guide.md` in line with these changes.